docs/source/advanced/multi-task-learning.rst

```diff
@@ -0,0 +1,266 @@
+                        "split_pct":	[0.8, 0.1, 0.1],
+                        "mask_field_names": ["train_mask_lp",
+                                             "val_mask_lp",
+                                             "test_mask_lp"]
+                    }
+                ]
+            },
+
+        ......
+
+        ]
+    }
+
+In the above configuration, we define one task for the **paper** node and one task for the
+**paper,citing,paper** edge. The node classification task will take the label name of `label_class` and the train/validation/test mask fields as `train_mask_class`,
+`val_mask_class` and `test_mask_class`, respectively. The link prediction task will take the train/validation/test mask fields as `train_mask_lp`, `val_mask_lp` and `test_mask_lp`, respectively.
+
+
+Construct Graph
+~~~~~~~~~~~~~~~~
+You can follow the instructions in :ref:`Run graph construction<run-graph-construction>` to use the
+GraphStorm construction tool for creating partitioned graph data for training. Please ensure you
+customize the command line arguments such as `--conf-file`, `--output-dir`, `--graph-name` to your
+specific values.
+
+
+Run Multi-task Learning Training
+--------------------------------
+Running a multi-task learning training task is similar to running other GraphStorm built-in tasks as
+detailed in :ref:`Launch Training<launch-training>`. The main defines is to define multiple training
+targets in the YAML configuration file.
+
+
+Define Multi-task for training
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can specify multiple training tasks for a training job by providing the `multi_task_learning`
+configurations in the yaml file The following config defines two training tasks, one for node
+classification and one for edge classification.
+
+.. code-blocks:: yaml
+
+    ---
+    version: 1.0
+    gsf:
+        basic:
+            ...
+        ...
+        multi_task_learning:
+            - node_classification:
+                target_ntype: "movie"
+                label_field: "label"
+                mask_fields:
+                    - "train_mask_field_nc"
+                    - "val_mask_field_nc"
+                    - "test_mask_field_nc"
+                task_weight: 1.0
+            - edge_classification:
+                target_etype:
+                    - "user,rating,movie"
+                label_field: "rate"
+                mask_fields:
+                    - "train_mask_field_ec"
+                    - "val_mask_field_ec"
+                    - "test_mask_field_ec"
+                task_weight: 0.5 # weight of the task
+
+Task specific hyperparameters in multi-task learning are same as those for single task learning as
+detailed in :ref:`Training and Inference<configurations-run>`, except that two new configs are required,
+i.e., `mask_fields` and `task_weight`. The `mask_fields` provides the specific training, validation and
+test masks for a task and the `task_weight` gives its loss weight.
```

Comment:
"gives its loss weight" -> "defines the weight value to be multiplied to a task's loss value when summating all task losses to compute the total loss for training."?